src/traffic_server/InkAPI.cc

```diff
@@ -6178,86 +6173,112 @@ TSHttpArgIndexNameLookup(UserArg::Type type, const char *name, int *arg_idx, con
   return TS_ERROR;
 }
 
+// -------------
+void
+TSUserArgSet(void *data, int arg_idx, void *arg)
+{
+  if (nullptr != data) {
+    PluginUserArgsMixin *user_args = dynamic_cast<PluginUserArgsMixin *>(static_cast<Continuation *>(data));
```

Comment:
If you want to make this more robust, check that `user_args` is not `nullptr` and `ink_assert` if it is, then
ignore it.

Reply:
Done.